src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -96,23 +96,6 @@ public void get( final String url ) {
 		check( "get", null );
 	}
 
-	@Override
-	public void close() {
```

Comment:
From the problem description I would assume that it suffices to remove the `close` method and keep the `quite` method...

Reply:
I should have mentioned this, but my thought was to have consistency. Since you can call `driver.close()` and the last window will quit the driver (according to documentation). But doing so does not call the `driver.quit()` method.

But as mentioned, I can do that, just wanted to share my thought process...

Reply:
Hmm. Additional calls to `re.cap()` are not hurtful, right. So I would think we should have it with `driver.quit()`, even if not with `driver.close()`.

Reply:
What do you think?

Reply:
After thinking about it, I would like to keep the proposed changes, i.e. separate both lifecycles. Since you still have to call other lifecycle methods (e.g. `startTest`, `capTest`; even though some are called transparently), why not also force users to call `cap` (assuming they do not want to run into the cap warner). 

Furthermore, we want users to use the test extensions, so that it manages the lifecycle for them (and we can transparently change some detail stuff in the background, e.g. call cap after all tests instead of just each test). 

Lastly, the `cap` method does IO (i.e. saving a report, uploading reports) which we do not want to do twice (especially uploading). But ultimately, we want to get rid of the `cap` method and upload only a changeset after each test.